nchelpers/__init__.py

```diff
@@ -725,7 +725,7 @@ def is_multi_year_mean(self):
         # Additional non-strict heuristics begin here
 
         # Heuristic: Time variable has "suspicious" length:
-        # 1, 4, 12, 5, 13, 16, 17 (yearly, seasonal, monthly, and
+        # 1, 4, 12, 365, 5, 13, 16, 17 (yearly, seasonal, monthly, daily, and
```

Comment:
No concatenations of daily with the others?

Reply:
I think it's safe to assume we won't be doing any concatenated timestamps in the future, because the version ncWMS we now use throws an error on non-monotonic timestamps, and concatenated datasets go from start to end of the year at one time resolution, then repeat for the next resolution, so there are always nonmonotonic timestamps involved. We had to un-concatenate some PDP datasets as part of that ncWMS upgrade. There are no longer any concatenated datasets in the database.

That said, aside from practicality concerns around ncWMS, concatenated datasets _are_ legal under the PCIC metadata standards, so maybe we should support them here. Thoughts?

Reply:
It strikes me that maybe we need to update the metadata standard to say "no concatenated climatologies." Not sure if this would cause any real-world problems (e.g., if scientists are still producing them, and we are separating the files ourselves). 

I would say if in practice we don't encounter them, then don't code to handle them. That leaves us with the question of what to do with the code that does handle certain concatenations. Remove it? Comment it to observe it is a legacy we are not continuing?

Reply:
I added a note to the Confluence metadata standards document that concatenated climatologies were obsolete.